packages/terra-core-docs/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 * Added
   * Added Accordion Examples for `terra-section-header`.
-  * Updated `terra-section-header` tests and examples.
+  * Updated `terra-section-header` tests and examples
```

Comment:
```suggestion
  * Updated `terra-section-header` tests and examples.
```

Reply:
Changed in https://github.com/cerner/terra-core/pull/3680/commits/33b97e65c97a86d2607cc471fe65233c7cc82a22